src/Components/Insights.js

```diff
@@ -0,0 +1,50 @@
+import '../Styles/Insights.css';
+
+import React, { Component } from 'react';
+import ReactApexChart from 'react-apexcharts';
+import { useState } from 'react';
+import Navbar from './Navbar';
+import { useRecoilValue } from 'recoil';
+import CuisineStats from '../recoils/CuisineStats';
+const Insights = () => {
+    const cuisines = useRecoilValue(CuisineStats);
+    const [options, setOptions] = useState({
+        chart: {
+            width: 380,
+            type: 'pie',
+        },
+        labels: ['Indian', 'Chinese', 'Mexican', 'Turkish'],
```

Comment:
make this generic

Reply:
and, move the default value out of the component and use it here.